src/main/java/com/faforever/api/security/FafUserDetailsService.java

```diff
@@ -23,6 +24,10 @@ public FafUserDetailsService(UserRepository userRepository) {
 
   @Override
   public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
-    return new FafUserDetails(userRepository.findOneByLoginIgnoreCase(username));
+    User uname = userRepository.findOneByLoginIgnoreCase(username); 
```

Comment:
Name the variable `user` instead of `uname`, since it contains a user and not a user name.